### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by pulchoilya**

### CODE REVIEW

File: src/checkPassword.test.js [hunk 1]
```diff
@@ -8,12 +8,53 @@ describe(`Function 'checkPassword':`, () => {
...
+    expect(typeof result).toBe('boolean');
   });
 
   it(`should return 'true' for the valid password with 8 characters`, () => {
+    expect(checkPassword('Pass1!aa')).toBe(true);
+  });
+
+  it(`should return 'true' for the valid password with 8 characters`, () => {
+    expect(checkPassword('Pass1!aa')).toBe(true);
+  });
```
Comment (4ksandrey) [inline src/checkPassword.test.js]: remove duplicate

File: src/checkPassword.test.js [hunk 2]
```diff
@@ -8,12 +8,53 @@ describe(`Function 'checkPassword':`, () => {
...
+
+  it(`should return 'true' for the valid password with 16 characters`, () => {
+    expect(checkPassword('Password1!AaBbCc')).toBe(true);
+  });
+
+  it('should return false for a password shorter than 8 characters', () => {
```
Comment (4ksandrey) [inline src/checkPassword.test.js]: specify boundary value in this test) 7 chars

File: src/checkPassword.test.js [hunk 3]
```diff
@@ -8,12 +8,53 @@ describe(`Function 'checkPassword':`, () => {
...
+    expect(checkPassword('Pass1!a')).toBe(false);
+  });
+
+  it('should return false for a password longer than 16 characters', () => {
```
Comment (4ksandrey) [inline src/checkPassword.test.js]: specify boundary value (17 chars)

File: src/checkPassword.test.js [hunk 4]
```diff
@@ -8,12 +8,53 @@ describe(`Function 'checkPassword':`, () => {
...
   });
 
-  // write more tests here
+  it('should return false for a password without an uppercase letter', () => {
+    expect(checkPassword('password1!')).toBe(false);
+  });
+
+  it('should return false for a password without a lowercase letter', () => {
+    expect(checkPassword('PASSWORD1!')).toBe(false);
+  });
```
Comment (4ksandrey) [inline src/checkPassword.test.js]: no requirements that 1 lowercase is required) revise it and fix

File: src/checkPassword.test.js [hunk 5]
```diff
@@ -8,12 +8,53 @@ describe(`Function 'checkPassword':`, () => {
...
+
+  it('should return false for a password containing a space', () => {
+    expect(checkPassword('Password 1!')).toBe(false);
+  });
+
+  it('should return false for a password'
+    + 'containing Cyrillic characters', () => {
+    expect(checkPassword('Password1!А')).toBe(false);
+  });
```
Comment (4ksandrey) [inline src/checkPassword.test.js]: fix test description (don't use concatenate) and provide input example where will be visible that this is Cyrillic letters)

**Review by 4ksandrey — CHANGES_REQUESTED** [review]
Check the comments

**Comment by pulchoilya** [comment]
It seems like everything was corrected, nothing was missed)))

**Review by Julia14012000 — APPROVED** [review]
Good job!
